rest/data/project_settings.go

```diff
@@ -551,6 +510,67 @@ func SaveProjectSettingsForSection(ctx context.Context, projectId string, change
 	return &res, errors.Wrapf(catcher.Resolve(), "saving section '%s'", section)
 }
 
+// getUnredactedSubscriptions parses the new subscriptions for any values that are redacted and if they are redacted,
+// it replaces the redacted value with the previous value for that subscription.
```

Comment:
nit: "it replaces the <REDACTED> placeholder string with the unredacted value" for clarity.